### DISCUSSION TITLE
Support launching custom URL's from within Charmstone

### DISCUSSION THREAD

**Discussion by NeighNeighNeigh** [rxhanson/Charmstone-Community#27] [Ideas]
Would be nice to add some custom URL's to my Charmstone launcher. I'm thinking "obsidian://…" "rectangle://…" etc. I know I can build an automator app with an `open -g "rectangle://…"` script (which I will do right now), but native support would be a nice feature if it's possible within the limitations of the sandbox?

<img width="824" alt="image" src="https://user-images.githubusercontent.com/4795315/223330301-32a66b1a-3316-4bd4-a610-921fb62ab594.png">

**Comment by rxhanson** [comment]
This has been on my todo list for quite some time. It is possible within sandbox. I'll see if I can get to it in the next release.
